core/src/main/java/org/renjin/primitives/packaging/Namespace.java

```diff
@@ -96,7 +94,7 @@ public String getCompatibleName() {
   }
 
   public SEXP getEntry(Symbol entry) {
-    SEXP value = namespaceEnvironment.getVariable(entry);
+    SEXP value = namespaceEnvironment.getVariableUnsafe(entry);
```

Comment:
indeed active bindings are not maintained as active binding in package namespace. This is now tested in evalOrder package.